[docs] CDCSDK: Documentation for CDC Consistent Snapshot streams (#21416)
yugabyte/yugabyte-db: docs/content/preview/admin/yb-admin.md

    create_change_data_stream ysql.yugabyte
```

> #### Creating a stream for Transactional CDC

Reviewer: ```suggestion
##### Creating a stream for Transactional CDC
```

> Creates a change data capture (CDC) DB stream for the specified namespace that can be used for Transactional CDC.

Reviewer: ```suggestion
Create a change data capture (CDC) DB stream for the specified namespace that can be used for Transactional CDC using the following command.

```

> This feature is {{<badge/tp>}}. The `yb_enable_cdc_consistent_snapshot_streams` flag can be used to enable the feature.

Reviewer 2: ```suggestion
This feature is {{<badge/tp>}}. Use the [yb_enable_cdc_consistent_snapshot_streams](../../reference/configuration/yb-tserver/#yb-enable-cdc-consistent-snapshot-streams) flag to enable the feature.

```

Author: Yes, this feature is in 2.20.2. Will backport these changes to stable pages as well

**Syntax**

```sh
yb-admin \
    -master_addresses <master-addresses> \
    create_change_data_stream ysql.<namespace_name> EXPLICIT CHANGE USE_SNAPSHOT
```

* *master-addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
* *namespace_name*: The namespace on which the DB stream ID is to be created.
* `EXPLICIT`: Checkpointing type on the server.

> * `CHANGE`: Record type indicating the server that the stream should send only the new values of the changed columns.

Reviewer: ```suggestion
* `CHANGE`: Record type indicating to the server that the stream should send only the new values of the changed columns.
```

> * `USE_SNAPSHOT`: Snapshot option indicating intention of client to consume the snapshot. If the client is not interested in consuming the snapshot, the `NOEXPORT_SNAPSHOT` option is to be used.

Reviewer: ```suggestion
* `USE_SNAPSHOT`: Snapshot option indicating intention of client to consume the snapshot. If the client is not interested in consuming the snapshot, use the `NOEXPORT_SNAPSHOT` option.
```
